src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -16,12 +16,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
```

Comment:
I will submit a follow-up PR that I have changes for locally. Do we need/want to do that for all the other packages as well?